CHANGELOG.md

```diff
@@ -43,9 +42,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Hide the Test MultiworkerSettings and GridStrategy classes from displaying in the editor. These are meant to only be used in Tests.
 - Reserved entity IDs previously expired after 3 minutes. Reserved Entity IDs now no longer expire, and persist until used.
 - A test was calling `SetReplicates` on an actor over which it did not have authority. This was causing warnings to be triggered. We've fixed this by reverting the actor's role at the end of the test, so that the actor is not left in an unexpected state.
-- Added support for clients to disconnect during a test in the automated test framework. 
+- Added support for clients to disconnect during a test in the automated test framework.
 - Modified ActorSystem's Ownership and Simulated Subviews to take player ownership into account.
 
+## [`0.13.1`] - 2021-06-02
+
+### Breaking changes:
+- Event tracing has been optimised to reduce overhead when tracing events in general and in particular when events are not sampled. The tracing API has been modified to accommodate these improvements. You will have to modify your project if you use the API.
+
+### Features:
+- Added `SpatialExecServerCmd` console command with one command `StartInsights` to dynamically enable insights capturing.
+  - Format: SpatialExecServerCmd < server > < command > < args >
+  - Example usage: "SpatialExecServerCmd local StartInsights -trace=CustomChannel -tracefile=MyNewTrace"
+- Visual Logger now supports multi-worker environments.
```

Comment:
Looks like this is a dupe 
![image](https://user-images.githubusercontent.com/31517089/121359448-66c69880-c92b-11eb-9dfc-f05d46485d22.png)

Reply:
Was done in both master and 13.1 so its an honest dupe I guess..

Reply:
Yeah fair enough